pkg/services/authz/zanzana/store.go

```diff
@@ -0,0 +1,13 @@
+package zanzana
+
+import (
+	"github.com/openfga/openfga/pkg/storage"
+	"github.com/openfga/openfga/pkg/storage/memory"
+)
+
+// FIXME(kalleep): Add support for postgres, mysql and sqlite data stores
+// Postgres and mysql is already imlemented by open fga so we need to add an implementation
+// for sqlite3. We could contribute that one upstream
```

Comment:
https://github.com/openfga/openfga/pull/1615

Reply:
Nice. Then I will just use memory for now when sqlite is configured and we can update it when it gets merged upsteam!